HubTurboUpdate.json

```diff
@@ -0,0 +1,12 @@
+{
+  "listOfLinks": [
```

Comment:
This is a generic list of downloadable HubTurbo link, whether user have downloaded it or not.

Use case:
for incremental data migration, list of links can have 1 link to version major 3, 1 link to version major 4 and 1 link to version major 5. Depending on user's version, they will download one with version major just 1 greater than their current version, e.g. in V3.4.1, user will update to V4.6.1 before V5.2.1.

Will rename the variable accordingly.